ruma-common/src/push.rs

```diff
@@ -80,6 +85,97 @@ impl Ruleset {
             AnyPushRule::Sender(r) => self.sender.insert(r),
         }
     }
+
+    /// Get the push actions that apply to this event.
+    ///
+    /// Returns `None` if no push action applies.
+    ///
+    /// # Arguments
+    ///
+    /// * `event` - The raw JSON of a room message event.
+    /// * `context` - The context of the message and room at the time of the event.
+    pub fn get_actions<'a, T>(
+        &'a self,
+        event: &Raw<T>,
+        context: &PushConditionRoomCtx,
+    ) -> Option<impl Iterator<Item = &'a Action>> {
+        let event_map = &flatten_json(event);
+
+        for rule in self.override_.iter().filter(|r| r.enabled) {
+            if rule.applies(event_map, context) {
+                return Some(rule.actions.iter());
+            }
+        }
+        for rule in self.content.iter().filter(|r| r.enabled) {
+            let condition = PushCondition::EventMatch {
+                key: "content.body".to_string(),
```

Comment:
Can you change all `"string literal".to_string()`s to `"string literal".into()` (or `"string literal".to_owned()` if inference fails for the former)? Just a small code style thing.